internal/service/alarms/internal/db/migrations/000003_create_alarm_event_record_table.up.sql

```diff
@@ -44,7 +44,7 @@ CREATE OR REPLACE FUNCTION update_alarm_event_sequence()
     RETURNS TRIGGER AS $$
 BEGIN
     -- Update sequence if status changes to 'resolved' or if alarm_changed_time is updated
-    IF (NEW.status = 'resolved' AND OLD.status IS DISTINCT FROM 'resolved')
+    IF (NEW.alarm_status = 'resolved' AND OLD.alarm_status IS DISTINCT FROM 'resolved')
```

Comment:
If we're going to be referencing these literals (e.g., 'resolved') frequently then it may be best to define some constants in a file specific to the alertmanager.

Reply:
hmmm you mean something like this https://github.com/openshift-kni/oran-o2ims/blob/main/internal/service/alarms/api/generated/alarms.generated.go#L38-L39 ?